jobs/integration/validation.py

```diff
@@ -785,209 +780,6 @@ def validate_status_messages(model):
             await asyncio.sleep(5)
 
 
-@log_calls_async
-async def validate_docker_logins(model):
```

Comment:
The docker-logins config isn't so much a docker runtime thing, as it is a registry credential thing. What's happening to this config? What is replacing it?

If I have a private registry, and I need to provide a username and password for it, how do I do that when I'm using the new docker subordinate? How do I do it with the new containerd subordinate?

Reply:
@Cynerva this has moved to the subordinate charm itself: https://github.com/juju-solutions/charm-docker/pull/1/files.  It'll need adding to the containerd subordinate, but will have to be tested on a per-subordinate basis I guess?

Reply:
Yeah, I don't think we want to lose this test. I'd prefer we update it to use the new config on docker if it's present.

Reply:
I was just reminded that the plan is to remove engine-specific stuff here, but reintroduce this test elsewhere. That works for me.